jmxtrans-output/jmxtrans-output-influxdb/src/main/java/com/googlecode/jmxtrans/model/output/InfluxDbWriter.java

```diff
@@ -227,5 +253,9 @@ public void doWrite(Server server, Query query, Iterable<Result> results) throws
 		return resultTagMap;
 
 	}
+	
+	private boolean isValidValue(Object value) {
+		return isValidNumber(value) || allowStringValues && value instanceof String;
```

Comment:
There are 2 changes in this PR: attributesAsTags and allowStringValues
Can you split this PR?

Reply:
allowStringValues was already in a independant PR : #681 
It was already merged early July.